app/mailers/notifier.rb

```diff
@@ -43,9 +44,8 @@ def invite(email, message, inviter, invitation_code, locale)
     @locale = locale
     @invitation_code = invitation_code
 
-    mail_opts = {:to => email, :from => AppConfig.mail.sender_address,
-                 :subject => I18n.t('notifier.invited_you', :name => @inviter.name),
-                 :host => AppConfig.pod_uri.host}
+    mail_opts = {:to => email, :from => AppConfig.mail.sender_address, :sender => AppConfig.mail.sender_address,
+                 :subject => I18n.t('notifier.invited_you', :name => @inviter.name)}
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.